deltachat-rpc-client/src/deltachat_rpc_client/rpc.py

```diff
@@ -0,0 +1,92 @@
+import asyncio
+import json
+import logging
+import os
+
+import aiohttp
+
+
+class JsonRpcError(Exception):
+    pass
+
+
+class Rpc:
```

Comment:
would be nice to point in the class docstring to a place where user can check for available API method names since they can't be auto-completed and are called dinamically